qiskit/backends/ibmq/ibmqprovider.py

```diff
@@ -39,14 +40,46 @@ def __init__(self, token, url,
     def get_backend(self, name):
         return self.backends[name]
 
-    def available_backends(self, filters=None):
+    def available_backends(self, *filters):
+        """Get a list of available backend instances from this provider.
+
+        Filters can be applied to get a subset of available backends.
```

Comment:
Can you tweak this docstring so `*filters` is included as an `Args: ...` entry?

Reply:
yep.